themes/default/content/docs/reference/service-rest-api/_index.md

```diff
@@ -1679,6 +1679,247 @@ Status: 204 OK
+
+```
+Status: 201 CREATED
+```
+
+```
+{
+  "organizationName":"demo",
+  "name":"bd7e0a35",
+  "displayName":"My Webhook",
+  "payloadUrl":"https://yourwebhook.com",
+  "active":true
+}
+```
+
+### List Webhooks
+
+```
+GET /api/orgs/{organization}/hooks
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name|
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+8' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token $PULUMI_ACCESS_TOKEN' \
+  https://api.pulumi.com/api/orgs/{organization}/hooks
+```
+
+#### Default response
+
+```
+[
+  {
+    "organizationName":"{organization}",
+    "name":"4e662b3b",
+    "displayName":"MyWebhook",
+    "payloadUrl":"http://mywebhook.com",
+    "active":true
+  },
+  {
+    "organizationName":"{organization}",
+    "name":"7732dd4c",
+    "displayName":"My secret webhook",
+    "payloadUrl":"https://mysecretwebhook.com",
+    "active":true
+  }
+]
+```
+
+### Get Webhook
+
+```
+GET /api/orgs/{organization}/hooks/{webhookname}
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `webhookname` | string | path | webhook name |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+8' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token $PULUMI_ACCESS_TOKEN' \
+  https://api.pulumi.com/api/orgs/{organization}/hooks/{webhookname}
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "organizationName":"{organization}",
+  "name":"{webhookname}",
+  "displayName":"My Webhook",
+  "payloadUrl":"http://mywebhook.com",
+  "active":true
+}
+```
+
+### Delete Stack
+
+```
+DELETE /api/stacks/{organization}/{project}/{stack}
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `webhookname` | string | path | webhook name |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+8' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token $PULUMI_ACCESS_TOKEN' \
+  --request DELETE \
+  https://api.pulumi.com/api/orgs/{organization}/hooks/{webhookname}
+```
+
+#### Default response
+
+```
+Status: 204 OK
+```
+
+```
+EMPTY RESPONSE BODY
+```
+
+### Ping Webhook
+
+```
+POST /api/orgs/{organization}/hooks/{webhookname}/ping
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `webhookname` | string | path | webhook name |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+8' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token pul-abcdefghijklmnopqrstuvwxyz' \
```

Comment:
```suggestion
  -H 'Authorization: token $PULUMI_ACCESS_TOKEN' \
```

Reply:
That's annoying - I thought I'd done through them all and changed them 🤦